internal/capability/auditor.go

```diff
@@ -43,20 +42,20 @@ func (capAuditor *capAuditor) BuildWorkQueueByCatalog(options operator.OperatorC
 	}
 
 	// Getting subscription data form the package manifests available in the selected catalog
-	subscriptions, err := c.GetSubscriptionData(options)
+	s, err := c.GetSubscriptionData(capAuditor.CatalogSource, capAuditor.CatalogSourceNamespace, capAuditor.FilterPackages)
```

Comment:
i think subscriptions reads better than s

Reply:
Cool. Just pushed the change.